h/browser/chrome/karma.config.js

```diff
@@ -51,7 +63,7 @@ module.exports = function(config) {
 
 
     // web server port
-    port: 9876,
+    port: 9877,
```

Comment:
Our other Karma tests (the ones for the Angular app) run on port 9876, which means that if you start these tests at the same time they will actually end up on port 9877 (but if you start them on their own they'll be on 9876).. Since I now need to know the port number for the proxy above, I had to fix this to make it constant.